samples/bluetooth/periodic_sync/src/main.c

```diff
@@ -246,8 +298,13 @@ void main(void)
 			err = bt_le_per_adv_sync_delete(sync);
 			if (err) {
 				printk("failed (err %d)\n", err);
-				return;
+				continue;
 			}
+			printk("success.\n");
+
+			/* Restore sync_count */
+			atomic_set(&sync_count, sync_count_prev);
```

Comment:
Ok, this will not work, if many already synchronized would terminate then the `sync_count` will be restored to incorrect value.